src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/ASEReadCounter.java

```diff
@@ -269,30 +270,18 @@ else if (base.getBase() == altAllele) {
         }
 
         final StringBuilder line = new StringBuilder();
-        line.append(pileup.getLocation().getContig());
-        line.append(separator);
-        line.append(pileup.getLocation().getStart());
-        line.append(separator);
-        line.append(siteID);
-        line.append(separator);
-        line.append(refAllele);
-        line.append(separator);
-        line.append(altAllele);
-        line.append(separator);
-        line.append(refCount);
-        line.append(separator);
-        line.append(altCount);
-        line.append(separator);
-        line.append(totalNonFilteredCount);
-        line.append(separator);
-        line.append(lowMAPQDepth);
-        line.append(separator);
-        line.append(lowBaseQDepth);
-        line.append(separator);
-        line.append(rawDepth);
-        line.append(separator);
-        line.append(otherBasesCount);
-        line.append(separator);
+        line.append(pileup.getLocation().getContig()).append(separator);
```

Comment:
So...  I think I gave you bad advice. It's very likely that the compiler would optimize the `+` version of this into exactly the same as the `StringBuilder`.  It wouldn't necessarily do so in a more complex expression, but it's very likely to do so in this case.  

http://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java